src/EventListener/PostQueryBuilderSubscriber.php

```diff
@@ -115,21 +117,28 @@ protected function applyRequestFilters(QueryBuilder $queryBuilder, array $filter
     protected function applyFormFilters(QueryBuilder $queryBuilder, array $filters = array())
     {
         foreach ($filters as $field => $value) {
+
             $value = $this->filterEasyadminAutocompleteValue($value);
             // Empty string and numeric keys is considered as "not applied filter"
             if (null === $value || '' === $value || \is_int($field)) {
                 continue;
             }
+
             // Add root entity alias if none provided
             $field = false === \strpos($field, '.') ? $queryBuilder->getRootAlias().'.'.$field : $field;
+            $property = $field;
+            if ($value instanceof ListFilter && $value->getProperty()) {
+                // if a property is specified in the ListFilter, it is on that property that we must filter
+                $property = $queryBuilder->getRootAlias().'.' .$value->getProperty();
```

Comment:
Prefixing with root alias is not only for ListFilter. It has to be done just after this if statement.

Reply:
which is what happens at lign 128, no ?

Maybe my mistake is not doing that test before the `$field` assignation though